src/ledger/InMemoryLedgerTxnRoot.cpp

```diff
@@ -10,8 +10,15 @@
 namespace stellar
 {
 
-InMemoryLedgerTxnRoot::InMemoryLedgerTxnRoot()
+InMemoryLedgerTxnRoot::InMemoryLedgerTxnRoot(
+#if BEST_OFFER_DEBUGGING
```

Comment:
do not mix `#if BEST_OFFER_DEBUGGING` and `#ifdef BEST_OFFER_DEBUGGING`
What you want everywhere is `#ifdef BEST_OFFER_DEBUGGING` given how you set it in the configure script

Reply:
Yikes, thanks for catching that. Fixed.